.github/workflows/jpa_integration_ci.yml

```diff
@@ -6,11 +6,9 @@ on:
   push:
     branches:
       - master
-      - 7.x
```

Comment:
How do this work?
Should the master branch's file mention all branches?
Or should each branch's file mention its own branch?

Reply:
Each branch should have its own .yaml file(s) with a filter for that particular branch. I think the multiple branches mentioned in the .yaml file on the master branch is a shortcut if you want to build multiple branches using the same workflow.

I would rather keep the branches separated. This test repo proves that triggering the CI on branches with different workflow definitions works: https://github.com/rsynek/github-actions-test/actions